pkg/controller/appdefinition/testdata/volumes/named-bound/expected.golden

```diff
@@ -101,7 +101,7 @@ metadata:
     acorn.io/app-name: app-name
     acorn.io/app-namespace: app-namespace
     acorn.io/managed: "true"
-    acorn.io/public-name: app-name.foo
+    acorn.io/public-name: other-app.foo
```

Comment:
This expected value changed because public names are now static and will not change when a new app binds an old volume.